Changes

```diff
@@ -1,5 +1,32 @@
 Release history for Zonemaster component Zonemaster-GUI 
 
+
+v3.4.0 2021-12-03 (public release version)
+
+ [Features]
+ - Replaces CentOS with Rocky Linux (#280)
+ - Provides translation to error messages from RPC API (#269, #239)
+ - Adds support from information banner (#266)
+ - Adds filter for test history (#253, #251)
+ - Adds run time configuration (#267, #260, #219, #170)
+ - Adds flexible URL (#252, #250)
+ - Updates Danish translation (#248, #244)
+ - Adds support for translated tooltips (#243, #241)
+
+ [Fixes]
+ - Updates Finnish translation (#290, #288)
+ - Updates Swedish translation (#286)
+ - Updates version of Node.js used (#274)
+ - Fixes translation issue in browsers (#278)
```

Comment:
Doesn't `in browsers` go without saying?

Reply:
I do not understand. I removed that item and moved `#278` to "Improves translation handling".